doc/src/modules/matrices/random.rst

```diff
@@ -0,0 +1,222 @@
+.. _matrices-random:
+
+Generate Specific Random Matrices
+=================================
+
+In order to setup distinct mathematical problems, like
+
+ - systems of linear equations,
+ - invertible matrices,
+ - triangular matrices
+ - differential equations with given eigenvalues
+ - linear isometries
+ - non-standard geometries by non-standard scalar products
+ - orthogonal basis of vector space,
+
+specific types of matrices are required.
+
+A purely random generation of matrix entries does not grant any
+specific shape or property of the matrix.
+So, those matrices have to be created carefully using
+classical classification theorems of linear algebra.
```

Comment:
This could reference the existing `randMatrix` function:
```python
In [4]: randMatrix(5)
Out[4]: 
⎡86  75  27  10  38⎤
⎢                  ⎥
⎢45  93  13  23  13⎥
⎢                  ⎥
⎢32  13  35  16  15⎥
⎢                  ⎥
⎢97  24  89  10  7 ⎥
⎢                  ⎥
⎣2   93  34  31  46⎦
```